.chloggen/elasticsearchexporter_otel-mode-log-body.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
As this is a breaking change in a Beta stability component, I believe it needs to be guarded by a feature gate.

Reply:
Oh actually it's a change in the `otel` mapping mode, which is experimental, so don't need a feature gate here.

Reply:
As discussed, OTel mode is experimental and this PR is aligned with the upcoming 8.16 ES mappings. It is "breaking" in nature but nothing will work without breaking it.